docs/public-networks/concepts/parallel-transaction-execution.md

```diff
@@ -0,0 +1,191 @@
+
+```mermaid
+graph TD;
+  A(Start) --> B(Fetch block's touched addresses);
+  B --> C{{Check each address}};
+  C -->|Unchanged| D(Mark as read);
+  C -->|Modified| E(Add to block's tracked addresses);
+  D --> F{{Next address}};
+  E --> F;
+  F -->|More addresses?| C;
+  F -->|No more| G(Fetch transaction's touched addresses);
+  G --> H{{For each transaction address}};
+  H -->|From, sender, etc.| I(Add to transaction's tracked addresses);
+  I --> J{{Next address}};
+  J -->|More addresses?| H;
+  J -->|No more| K{{Compare block and transaction addresses}};
+  K -->|Conflict detected| L(Conflict is detected);
+  K -->|No conflict| M(Proceed with transaction);
+  L --> N(End);
+  M --> N;
+```
+
+</p>
+
+Besu's conflict detection strategy is intentionally simple to minimize edge cases.
+With this approach to parallel transaction execution,
+[approximately 40% of transactions do not require replay](#metrics).
+In the future, the conflict detection strategy may be refined to reduce false positives.
+
+You can enable parallel transaction execution using the `--Xbonsai-parallel-tx-processing-enabled` option.
+
+## Metrics
+
+Parallel transaction execution uses Besu's resources more efficiently than traditional
+sequential execution, significantly improving performance.
+
+The following metrics were collected on nodes running on Azure VMs (Standard D8as v5 – 8 vCPUs, 32
+GiB memory), with Teku and Nimbus as consensus layer (CL) clients:
+
+- **Block processing time** - With Teku as CL client, block processing time improves by at least 25%.
+  The 50th percentile decreases from 282 ms to 207 ms and the 95th
+  percentile decreases from 479 ms to 393 ms.
+
+  With Nimbus as CL client, block processing improves by approximately 45%, with the 50th percentile
+  at 155 ms, and the 95th percentile at 299 ms.
+  This is likely due to both the EL and CL clients running on the same machine, which reduces cache
+  misses and context switching.
+
+- **Execution throughput** - Execution throughput increases, with an average of 96 Mgas/s and peaks
+  of up to 25 Mgas/s.
+
+- **Parallel transactions** - Parallel transaction execution introduces two new metrics, which
+  indicate that approximately 40% of transactions are parallelized using this feature:
+
+  - `besu_block_processing_parallelized_transactions_counter_total` - The number of transactions
+    executed in parallel.
+  - `besu_block_processing_conflicted_transactions_counter_total` - The number of transactions that
+    encountered conflicts and were therefore executed sequentially.
+
+- **Sync time** - Snap synchronization time is approximately 27 hours and 5 minutes, with block import
+  time approximately 6 ms on average.
+
+- **CPU profiling** - The new payload call time decreases from 251.68 ms to 172.04 ms on average,
+  with notable improvements in SLOAD operation times.
+
+Overall, parallel transaction execution improves Besu performance with almost no resource usage
+overhead.
```

Comment:
We can add here : During block processing time, 172.04 ms on average for our case for each 12 seconds slot time, the node will use more CPU and more disk accesses in parallel (higher IOPS), but when these metrics are averaged on different monitoring tools, the resources usage looks the same as before.